pkg/api/server/v1alpha2/results.go

```diff
@@ -26,6 +27,8 @@ func (s *Server) CreateResult(ctx context.Context, req *pb.CreateResultRequest)
 
 	// Populate Result with server provided fields.
 	r.Id = uid()
+	r.CreatedTime = timestamppb.Now()
+	r.UpdatedTime = timestamppb.Now()
 
 	store, err := result.ToStorage(parent, name, req.GetResult())
```

Comment:
As is, I think this is going to strip out the timestamp in the ToStorage function, since we don't actually store this data anywhere in the model.

Let's use a [Clock](https://pkg.go.dev/github.com/jonboulle/clockwork) to make the timestamps deterministic and test to make sure they can be successfully retrieved.

Reply:
Great! Done :)